app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -701,18 +701,30 @@ export default {
         if (this.isSignatureEnabledForInbox && this.messageSignature) {
           newMessage += '\n\n' + this.messageSignature;
         }
-        const messagePayload = this.getMessagePayload(newMessage);
+
+        if (this.isWhatsAppChannel) {
+          this.sendMessageAsMultipleMessages(newMessage);
+        } else {
+          const messagePayload = this.getMessagePayload(newMessage);
+          this.sendMessage(messagePayload);
```

Comment:
This changes the sequence of things. Earlier `clearMessage` was called after `sendMessage`, now it's before. Is that fine?

Reply:
This was done as we are iterating over each attached message and sending it. Calling clearMessage would clear the `attachedFiles` state variable. Let me know your thoughts